Jump Search/jump_search.cpp

```diff
@@ -0,0 +1,76 @@
+#include<iostream>
+#include<cmath> //to use sqrt()
+using namespace std;
+int jump_search(int a[], int n, int key)
+{
+    int step=sqrt(n);
+    int pre_step=0;
+    while (a[min(step, n)-1] < key) 
+    { 
+        pre_step = step; 
+        step += sqrt(n); 
+        if (pre_step >= n)
+        {
+            return -1;	
+        } 
+    }
+    while(a[pre_step]<key)
+    {
+        pre_step++;
+        if(pre_step==min(step,n))
+        {
+            return -1;
+        }
+    }
+    if(a[pre_step]==key)
+    {
+        return pre_step;
+    }
+    else
+    {
+        return -1;
+    }
+}
+int main()
+{
+    int no,ele;
+    cout<<"Enter no. of elements in an array: ";
+    cin>>no;
```

Comment:
give space around operators throughout the file

Reply:
done!